pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1075,6 +1151,34 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 	doActivateTail(ctx, status, domainId)
 }
 
+func doActivateTailContainer(ctx *domainContext, status *types.DomainStatus,
```

Comment:
Having this function is probably fine for now, but once we start passing xl.cfg to rkt we'll be able to get rid of it an just reuse good ol' doActivateTail

Reply:
Agree.